request-node/values.yaml

```diff
@@ -67,6 +68,8 @@ ipfs:
     port: 4001
   # IPFS API port
   api:
+    # required if `ipfs.sidecar` is false
+    # host: 
```

Comment:
Would this work? It wasn't obvious to me at first glance that the comment above was applying to the "host" property, since host appears greyed out, I thought the comment was applying to "port".
```suggestion
    host: null
```

Reply:
better?

Reply:
Perfect  👌